src/pages/index.js

```diff
@@ -117,6 +148,17 @@ const Page = () => (
             <IndexCardLink to="/supported-open-source/barman/">
               Barman
             </IndexCardLink>
+
+            <IndexCardLink
+              to="/supported-open-source/barman/single-server-streaming/"
+              className="nested-link"
+            >
+              Single Server Streaming
+              <span className="new-thing" title="Interactive Katacoda Tutorial">
+                {' '}
```

Comment:
What's this accomplishing? Could padding or the `&:before content` replace this? Not a big deal, just curious

Reply:
I think the linter added that when I committed :(